tests/core/test_longjmp_i64.c

```diff
@@ -14,7 +16,7 @@
 jmp_buf jb;
 
 __attribute__((noinline)) void foo(int64_t x) {
-  printf("foo: %lld.\n", x);
+  printf("foo: %" PRId64 ".\n", x);
```

Comment:
This feels unrelated, weird that you had to do this. It seems more-correct but I'm wondering what happened there.

Reply:
That test won't compile for me without making that change.